checkov/terraform/checks/graph_checks/aws/IAMManagedIAMFullAccessPolicy.yaml

```diff
@@ -0,0 +1,42 @@
+---
+metadata:
+  id: "CKV2_AWS_51"
+  name: "Ensure AWS Managed IAMFullAccess IAM policy is not used."
+  category: "IAM"
+  severity: "high"
```

Comment:
```suggestion
```
severity is later added in the docs and setting it here has no relevance.

Reply:
removed